### PR TITLE
[new] add ion-rangeslider@2.0.13 with git auto-update, close #5247

### LINKED ISSUE #5247: [Request] Ion.Range Slider
**Library name: Ion.Range Slider
**Git repository url: https://github.com/IonDen/ion.rangeSlider
**npm package url(optional):** 
**Official homepage: http://ionden.com/a/plugins/ion.rangeSlider/en.html
**Wanna say something? Leave message here: It's a good range slider plugin

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by Amomo**
Hi @maruilian11 , I add a new lib with git auto-update from the request #5247
repo: https://github.com/IonDen/ion.rangeSlider
Would you mind checking this PR for me? Thank you~ :-)

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : **Watch `39`, Star `718`, Fork `187`**
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

### CODE REVIEW

**Comment by maruilian11** [comment]
@Amomo i think you also need to include `normalize.css` and its min file, and you had better add `css/` before those requiredFiles.

**Comment by Amomo** [comment]
@maruilian11 because `normalize.css` is come from another library, I think we shouldn't collect it here.
repo: https://github.com/necolas/normalize.css/
Thanks for your reminding about "requiredFiles", I have corrected it!
Could you take a look again? Thank you :-D

**Comment by maruilian11** [comment]
@Amomo thank you for your explaining.
@PeterDaveHello i think it's ok.

File: ajax/libs/ion-rangeslider/package.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    "source": "git",
+    "target": "git://github.com/IonDen/ion.rangeSlider.git",
+    "basePath": "",
+    "files": [
+      "js/ion.rangeSlider*.js",
+      "css/ion.rangeSlider*.css",
+      "img/sprite-skin*.png"
+    ]
+  },
+  "requiredFile": [
```
Comment (PeterDaveHello) [inline ajax/libs/ion-rangeslider/package.json]: wrong field name ...

**Comment by Amomo** [comment]
@maruilian11 I have corrected it!
Could you take check it again? Thank you~ 😀

**Comment by Amomo** [comment]
Hi @maruilian11 I have corrected it and rebased to latest master branch.
Could you take a look again? Thank you~ :-D

**Comment by maruilian11** [comment]
@PeterDaveHello i think it's ok.

**Pull request merged**